app/src/main/java/com/eveningoutpost/dexdrip/utils/DexCollectionType.java

```diff
@@ -59,13 +61,13 @@
             mapToInternalName.put(dct.internalName, dct);
         }
 
-        Collections.addAll(usesBluetooth, BluetoothWixel, DexcomShare, DexbridgeWixel, LimiTTer, WifiBlueToothWixel, DexcomG5, WifiDexBridgeWixel);
-        Collections.addAll(usesBtWixel, BluetoothWixel, LimiTTer, WifiBlueToothWixel);
-        Collections.addAll(usesWifi, WifiBlueToothWixel,WifiWixel,WifiDexBridgeWixel, Mock);
+        Collections.addAll(usesBluetooth, BluetoothWixel, DexcomShare, DexbridgeWixel, LimiTTer, WifiBlueToothWixel, DexcomG5, WifiDexBridgeWixel, LimiTTerWifi);
+        Collections.addAll(usesBtWixel, BluetoothWixel, LimiTTer, WifiBlueToothWixel, LimiTTerWifi); // Name is misleading here, should probably be using dexcollectionservice
```

Comment:
The `usesBtWixel` is to describe the simple text based protocol used by things like LimiTTer and the classic xDrip device. It doesn't include things like xBridge or blucon, miaomiao etc

Reply:
Also see below where we have a specific set for whether something uses dexcollection service

Reply:
Ok, so I know what got me confused:
The tomato, does not have a type of it's own, rather it was using the limiter name.

So, I guess that there are two solutions here:
1) Change the line to be:
   Collections.addAll(usesBtWixel, BluetoothWixel, LimiTTer, WifiBlueToothWixel, LimiTTerWifi); //  describe 
    the simple text based protocol used by things like LimiTTer and the classic xDrip device
I'll still add the LimiTTerWifi since this is still limiter.
2) Add a new HashSet<DexCollectionType> usesLimiterBT and add limite, limiterWifi there.

What do you think is the better solution?